mmv1/templates/terraform/examples/secure_source_manager_repository_basic.tf.erb

```diff
@@ -0,0 +1,21 @@
+resource "google_secure_source_manager_instance" "instance" {
```

Comment:
Is the example here also used as tests?

Reply:
Yep

Reply:
Can you add a couple more test cases? e.g. no init config and even failed cases. If these are also the examples displayed in the resource page like https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/secure_source_manager_instance, then failed cases might not be a good fit.

Reply:
Sure will do. Yea, it seems like these are all shown as examples, so you can't add failed cases.

Reply:
Added no initial config case